aospy_synthetic/region.py

```diff
@@ -0,0 +1,103 @@
+"""region.py: Region class and region_inst()."""
```

Comment:
Yes, nothing has been changed (I just took the latest version from aospy).  The other objects (Proj, Model, Run, Calc, etc.) are a bit dated, but the edits to add DB functionality are very small (admittedly in aospy synthetic, the core objects are basically there just to store metadata in a hierarchical way that mimics the way it works in practice -- perhaps I can be cleaner in removing all unnecessary methods).